website/content/en/status/report-2023-10-2023-12/_index.adoc

```diff
@@ -0,0 +1,159 @@
+---
+title: "FreeBSD Status Report Fourth Quarter 2023"
+sidenav: about
+---
+
+= Introduction
+:doctype: article
+:toc: macro
+:toclevels: 2
+:icons: font
+:!sectnums:
+:source-highlighter: rouge
+:experimental:
+:reports-path: content/en/status/report-2023-10-2023-12
+
+include::content/en/status/categories-desc.adoc[]
+
+include::{reports-path}/intro.adoc[]
+
+'''
+
+toc::[]
+
+
+'''
+
+[[FreeBSD-Team-Reports]]
+== FreeBSD Team Reports
+
+{FreeBSD-Team-Reports-desc}
+
+'''
+
+include::{reports-path}/core.adoc[]
+
+'''
+
+include::{reports-path}/freebsd-foundation.adoc[]
+
+'''
+
+include::{reports-path}/releng.adoc[]
+
+'''
+
+include::{reports-path}/ci.adoc[]
+
+'''
+
+include::{reports-path}/portmgr.adoc[]
+
+'''
+
+include::{reports-path}/bugmeister.adoc[]
+
+'''
+
+[[userland]]
+== Userland
+
+{userland-desc}
+
+'''
+
+include::{reports-path}/service-jails.adoc[]
+
+'''
+
+[[kernel]]
+== Kernel
+
+{kernel-desc}
+
+'''
+
+include::{reports-path}/packrat.adoc[]
+
+'''
+
+[[architectures]]
+== Architectures
+
+{architectures-desc}
+
+'''
+
+include::{reports-path}/armv7.adoc[]
+
+'''
+
+include::{reports-path}/simd.adoc[]
+
+'''
+
+[[cloud]]
+== Cloud
+
+{cloud-desc}
+
+'''
+
+include::{reports-path}/openstack.adoc[]
+
+'''
+
+include::{reports-path}/azure.adoc[]
+
+'''
+
+include::{reports-path}/ec2.adoc[]
+
+'''
+
+[[documentation]]
+== Documentation
+
+{documentation-desc}
+
+'''
+
+include::{reports-path}/doceng.adoc[]
```

Comment:
Not consistent with the actual filename, doceng-2023-4th-quarter-status-report.adoc.